pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -307,6 +307,24 @@ const (
 	FullyApplied string = "FullyApplied"
 )
 
+// These are reasons for a binding's transition to a Scheduled condition.
+const (
+	// BindingReasonSuccess reason in Scheduled condition means that binding has been scheduled successfully.
+	BindingReasonSuccess = "Success"
```

Comment:
Actually, I don't think we need a reason for successful condition. If a pod is scheduled to a node successfully, reason will not be set as well.

Reply:
> Actually, I don't think we need a reason for successful condition. If a pod is scheduled to a node successfully, reason will not be set as well.

I agree with you but it's required. BTW PodCondition overrides this field and marks it as optional.
![image](https://github.com/karmada-io/karmada/assets/89241565/376c259c-4d4d-4eae-ac0f-18326b8e772e)